### DISCUSSION TITLE
2021/12/2021-总结/

### DISCUSSION THREAD

**Discussion by giscus** [SouthFox-D/SouthFox-D.github.io#13] [博客评论]
# 2021/12/2021-总结/

是时候写个总结了。

https://blog.southfox.me/2021/12/2021-%E6%80%BB%E7%BB%93/

**Comment by SouthFox-D** [comment]
2023再见！

**Comment by yeung66** [comment]
这像素风的主题看起来好棒！

> 冲浪看了很多牛人的博客，冒出很多自愧不如的情感，那么多努力，我就有那么多惆怅……

这种现象其实很正常，我也常会这样觉得，对比于博友丰富的经历和所做的事情，自己似乎并没有做出什么。不过随着自己越看越多，似乎也开始变得坦然起来。

**Reply by SouthFox-D** [reply]
是的……随着看的人越多，也就越来越坦荡了。

博客视觉效果有一大半都是字体的功劳了hhh，我只是做了点微小的工作，把它们两个缝合了起来。
